packages/checkout/sdk/src/widgets/definitions/parameters/sale.ts

```diff
@@ -23,6 +24,8 @@ export type SaleWidgetParams = {
   walletProviderName?: WalletProviderName;
   /** The language to use for the sales widget */
   language?: WidgetLanguage;
+  /** The disabled payment types */
+  disabledPaymentTypes: SalePaymentTypes[];
```

Comment:
Just a note in terms of naming, for filtering wallets I'm using `includeWallets` and `excludeWallets` .. was just wondering out loud if it made sense here to align the language like `excludePaymentTypes` or `includePaymentTypes`?

Reply:
Yes, it does thank you!

Reply:
Done